packages/e2e/local-network/scripts/nodes-configuration.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+AMOUNT_PER_NODE='10000000000000'
+NUM_SPO_NODES=11
```

Comment:
```suggestion
NUM_SP_NODES=11
```
I know this is not a new variable but let's get this corrected now. SPO is an acronym for Stake Pool Operator.